packages/server/src/app.controller.ts

```diff
@@ -4,15 +4,19 @@ import config from './config';
 
 @Controller()
 export class AppController {
-
-  constructor(
-    private readonly appService: AppService,
-  ) {
-  }
+  constructor(private readonly appService: AppService) {}
```

Comment:
Is the empty constructor necessary?

Reply:
Yes, since Nestjs uses reflection on classes for dependency injection. Also, using modifiers such as `public` on a constructor property is a shorthand for declaring it as a class property, see https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties for details